crates/biome_js_analyze/src/lint/nursery/use_consistent_object_literals.rs

```diff
@@ -0,0 +1,189 @@
+use biome_analyze::{
+    context::RuleContext, declare_lint_rule, Ast, Rule, RuleDiagnostic, RuleSource, RuleSourceKind,
+};
+use biome_console::markup;
+use biome_deserialize_macros::Deserializable;
+use biome_js_syntax::{inner_string_text, AnyJsExpression, AnyJsObjectMember};
+use biome_rowan::AstNode;
+use serde::{Deserialize, Serialize};
+
+declare_lint_rule! {
+    /// Require consistently using either explicit object property assignment or the newer shorthand syntax, when possible.
```

Comment:
```suggestion
    /// Require the consistent declaration of object literals. Defaults to shorthand definitions.
```

What do you think of this shorter version?

Reply:
Sounds good. But personally I would set the default as explicit, it always felt weird seeing the shorthand syntax for me, especially because you can only use it in some cases. Meanwhile you can force explicit and it will always be possible and consistent.

Reply:
Sorry, I thought that the default was the shorthand. Feel free to correct my suggestion